content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,80 @@ menu:
 weight: 101
 ---
 
+## v2.0.4 General Availability [2021-02-04]
```

Comment:
I think we're going to end up with more additions here. The list I know of (hopefully there won't be any more):
* Changing the binary naming pattern
* Fixing a config-loading bug
* Adding the Flux dict expressions to Swagger

Reply:
influxdata/influxdb#20708
influxdata/influxdb#20709
influxdata/influxdb#20710

Reply:
added these